Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -296,6 +310,32 @@ private static void CheckOptimalConfiguration()
             }
         }
 
+        
+        /// <summary>
+        /// Helper method to update the a layer name at given index in the related project asset file
+        /// </summary>
+        /// <param name="layerNumber">index of layer to modify. Should be between 8 and 31</param>
+        /// <param name="layerName">Name of layer to set</param>
+        private static void CreateLayer(int layerNumber, string layerName)
```

Comment:
We should likely remove this functionality from MixedRealityToolkit (i believe that is were it currently resides)

Reply:
@davidkline-ms oh nice! didn't even know they already existed. I delete redundant code